### ISSUE TITLE
Performance Gap in Reproducing PTv3 on S3DIS Dataset

### ISSUE THREAD

**Issue by Fan-QY** [Pointcept/PointTransformerV3#160]
I have successfully reproduced PTv3 under S3DIS, following the procedures outlined in your paper. However, the final results are not ideal, with a nearly 4% mIoU difference compared to your work when training from scratch on the S3DIS dataset.

The best mIoU achieved during training was 0.6865 (val Folder 5 of S3DIS). However, after TTA, the results reached  0.7023. But in your work ist 0.7433/0.7343. Could you please review my training logs and environment configuration to identify any potential issues? What could be causing this discrepancy?

Thanks a lot!
Best regards

[train.log](https://github.com/user-attachments/files/21235666/train.log)

**Comment by yuqingsh** [comment]
Hi, I also reproduced PTv3 with 4 A800 GPUs, and got similar result as yours.

I then went checking the train log released here: https://huggingface.co/Pointcept/PointTransformerV3/tree/main/s3dis-semseg-pt-v3m1-0-rpe

I believe the 0.73 mIoU reported in the paper is trained using the config of rpe. However, this config does not allow using flash attention. I believe without flash attention, there will be CUDA OOM error.

**Comment by Fan-QY** [comment]
I reduced the learning rate by half on two A6000ada (48GB) and obtained good results (val 0.7077, test with TTA 0.7154), but I have not yet achieved the best results of the original work and I will try with rep config. The following is the log.

[train2.txt](https://github.com/user-attachments/files/21327994/train2.txt)
